webpack.config.js

```diff
@@ -142,17 +144,16 @@ module.exports = {
     "bootstrap": resolve(scriptPath, "bootstrap.js"),
   },
   output: {filename: "scripts/[name].js", path: staticPath},
-  optimization: {minimizer: [new TerserPlugin(), new OptimizeCssAssetsPlugin({})]},
+  optimization: {minimizer: ['...', new CssMinimizerPlugin()]},
   module: {
     rules: [{
       test: /\.scss$/,
       use: [
-        {loader: "file-loader", options: {name: "styles/[name].css"}},
-        {loader: "extract-loader"},
-        {loader: "css-loader?-url"}, //url()-inlining turned off}
+        {loader: MiniCssExtractPlugin.loader},
+        {loader: "css-loader?-url"}, //url()-inlining turned off
         {loader: "sass-loader",},
       ],
     }],
   },
-  plugins: [htmlWebpackPlugin, copyPlugin],
+  plugins: [htmlWebpackPlugin, copyPlugin, new MiniCssExtractPlugin()],
```

Comment:
I think this is what you need to fix the docs site:

```suggestion
  plugins: [htmlWebpackPlugin, copyPlugin, new MiniCssExtractPlugin({
    filename: "styles/[name].css"
  })],```